.github/workflows/ci.yml

```diff
@@ -17,9 +17,18 @@ jobs:
         version: "23.3"
         repo-token: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: Run clippy
+    - name: Install wasm Rust
+      uses: ATiltedTree/setup-rust@v1
```

Comment:
hmmm, that's an exotic pick :) I guess that's because the actions-rs got archived? I'd rather think about going with dtolnay's one in that case but it should also be then changed in other steps

Reply:
I'm fine with continuing to use actions-rs too tho until we decide what we migrate too, as there are also new players in the game like:
https://github.com/moonrepo/setup-rust

Reply:
It being archived scared me off, but it seems like it installs wasm clippy just fine, switched to `actions-rs` 👍